extendr-api/Cargo.toml

```diff
@@ -42,13 +42,14 @@ full-functionality = ["graphics", "either", "ndarray", "faer", "num-complex", "s
 # Parts of the R-API are locked behind non-API, as CRAN frowns upon the presence
 # of non-API items in packages. You may enable this feature, to generate
 # bindings to these non-API items, and their usage in extendr
-non-api = ["libR-sys/non-api"]
+# BEWARE: this feature does not work, as `bindgen` use in extendr is deprecated
+non-api = []
```

Comment:
@CGMossa this needs to be a no-op because we don't have a way to generate the bindings via build.rs for someone who wants to use `libR-sys/non-api` at the moment. We will need to address afterwards https://github.com/extendr/libR-sys/issues/251

Reply:
It can't be a no-op. It just breaks if you add it to the compilation process.